.make/test.mk

```diff
@@ -118,7 +118,7 @@ DOCKER_COMPOSE_BIN_ALT = $(TMP_PATH)/docker-compose
 DOCKER_COMPOSE_FILE = $(CUR_DIR)/.make/docker-compose.integration-test.yaml
 
 # This pattern excludes some folders from the coverage calculation (see grep -v)
-ALL_PKGS_EXCLUDE_PATTERN = 'vendor\|account\/tenant\|app\|tool\/cli\|design\|client\|test'
+ALL_PKGS_EXCLUDE_PATTERN = "vendor\|account\/tenant\|app\'\|tool\/cli\|design\|client\|test"
```

Comment:
Why single quote is escaped here?
Why forward slash is escaped after `account`? That should be a pipe character, right?

Reply:
@jarifibrahim Thanks for the clarification!